parser/Builder.cc

```diff
@@ -1742,7 +1742,10 @@ class Builder::Impl {
     }
 
     void checkDuplicateArgs(sorbet::parser::NodeVec &args, UnorderedMap<core::NameRef, core::LocOffsets> &map) {
+        int pos = -1;
         for (auto &this_arg : args) {
+            ++pos;
```

Comment:
This is the real reason why I want the ranges library.

Reply:
Oh me too, `ranges::enumerate` would be so much nicer here.